src/components/Home.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
+import { Button } from 'react-bootstrap';
 
 const Home = () => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="text-center">
+      <h1>{"CYF Quiz"}</h1>
+      <p>Hello Coder !</p>
+      <p>Are you ready for the next challenge?</p>
+      <p>Take a quiz!</p>
```

Comment:
These paragraphs look quite far apart from each other — let's make the spacing between them smaller so they're not as spread out

Reply:
yes, done that.